src/BusinessFoundationModules.code-workspace

```diff
@@ -2,15 +2,15 @@
 	"folders": [
 		{
 			"name": "No. Series",
-			"path": "Business Foundation/App/No Series"
+			"path": "Business Foundation\\App"
```

Comment:
well this is not correct... the name of the folder still says "No. Series" but now the path doesn't match.

also don't get why you're changing / to \\ ... / is nicer i think... 

Anywho... this workspace is BusinessFoundation_Modules_ so it is meant to show as
module1
module1testlib
module1test
module2
module2testlib
etc...

We might need a second workspace, BusinessFoundation.code-workspace which has
App
Test Library
Test

as you're doing here...